src/components/RepoValidation.js

```diff
@@ -147,6 +147,8 @@ function RepoValidation() {
         <CircularProgress size={180} />
       </>
       :
+      <AdminContextProvider>
```

Comment:
Should the AdminContextProvider go inside _app.js? Or is there a reason why you have it here?

Reply:
Done! The move also revealed a timing issue that was lurking in the code. So this was a great suggestion.